src/script/common/c_content.cpp

```diff
@@ -1315,12 +1315,12 @@ void pushnode(lua_State *L, const MapNode &n)
 }
 
 /******************************************************************************/
-void warn_if_field_exists(lua_State *L, int table,
-		const char *name, const std::string &message)
+void warn_if_field_exists(lua_State *L, int table, const char *fieldname,
+		const std::string &name, const std::string &message)
```

Comment:
this is a good opportunity to change name and message to `std::string_view`

Reply:
Applied in f7004fa75

Reply:
Seems like there is so much abuse to `std::string &` in place of `std::string_view`...